requirements_dev.txt

```diff
@@ -29,6 +29,7 @@
 mock>=1.3.0
 pylint==1.7.4
 pycodestyle==2.3.1
+pandas==0.22.0
```

Comment:
adding pandas as a dependency just to run a test seems like overkill

Reply:
I know but it's a only dev dependency so I don't think it's a big deal.
And that's the only way I can think of to test the expected behaviour

Reply:
If it is such a big deal, checking the type of the value might suffice.

Reply:
Ah wait, maybe I can check on a np.array instead. Let me try